docs/FormFillerButton.md

```diff
@@ -33,6 +33,23 @@ export const ContactEdit = () => (
 );
 ```
 
+You must define a `dataProvider.getCompletion()` method to fetch the completion suggestions from your API. This method must return a promise that resolves to a `{ data: completionString }` object.
```

Comment:
Since ra-ai v5.2.0 the FormFillerButton uses `dataProvider.generateContent()`

This also makes me realize the ra-ai doc is wrong, e.g. the section dedicated to `meta`: https://react-admin-ee.marmelab.com/documentation/ra-ai#meta-2